app/uk/gov/hmrc/tai/service/NuanceEncryptionService.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.service
+
+import com.google.inject.Inject
+import play.api.Configuration
+import uk.gov.hmrc.crypto.{CryptoGCMWithKeysFromConfig, PlainText, Scrambled, Sha512Crypto}
+
+/**
+  * Service for encrypting data to send to Nuance (Virtual Assistance)
+  * The configuration here (and algorithm) needs to match that in userid-recovery-api
+  * which contains the decryption code
+  */
+case class NuanceEncryptionService @Inject()(configuration: Configuration) {
+  protected val FIELD_PREFIX = "ENCRYPTED-"
```

Comment:
this protected makes me feel like we might be interested in inheriting this case class. but inheriting a case class doesn't make sense. I'd suggest removing the case from the case class or use a trait from which NuanceEncryptionService will inherit

also the name doesn't follow the scala convention